src/Client/Modals/SelectiveImportModal.fs

```diff
@@ -1,77 +1,24 @@
-namespace Modals
+namespace Modals.Import
```

Comment:
This is the only file in this exact namespace. Not required therefore.

Reply:
fixed